### DISCUSSION TITLE
BT Notification call to function of WinRT event

### DISCUSSION THREAD

**Discussion by fd1911** [Windos/BurntToast#118] [General]
Hello there!
I've been following this module for quite a wile and unfortunately I still cannot get the hang out of it.
I've created a script with two buttons and those right now open a website, because as far as I saw until 8.0 the buttons couldn't do more than this.

This is the actual code block inside a function. (a very simple if else)

`        $appid = New-BTAppId  -AppId 'NotificationAPP' `
`        $image= 'C:\Scrtipt\Avatar.jpg' `
`        $Button1 = New-BTButton -Content 'Remember submitting reports' -Arguments 'www.facebook.com'`
`        $Button2 = New-BTButton -Content 'Reports already loaded' -Arguments 'www.twitter.com'`
 `       $alarm = 'Alarm2'`
`        New-BurntToastNotification -applogo $image -Text 'Remember submitting reports' -Button $Button1,$Button2 -Sound $alarm`

I am kinda lost in the subject of making the button call to a function in the script or even launch an app.
Can someone be gentle in letting me know how can I try this?
I've already looked into the new documentation and I didn't understood it and tried to use the SubmitBT-Notification as well the BT Action module
(Maybe my English is a little bit rusty, sorry for that.)
Anyway, thanks a lot for creating this project!

Thanks!

**Comment by Windos** [comment]
Hey @fd1911! This is the only thing I've had time to write documentation wise on putting "real" actions behind buttons, [Actionable Toast Notifications, Finally Cracked Them!](https://toastit.dev/2020/08/08/actionable/).

I've also got a [video from a livestream](https://www.youtube.com/watch?v=8TuqVs7QGz0) when I worked on v0.8.0, which shows slightly newer way of doing this (supplying a **ScriptBlock** to **$ActivatedAction** on `Submit-BTNotification` or `New-BurntToastNotification`)

What you're going to need to work out, is which button was pressed, in my examples so far you could technically select an item from the drop down box and press anywhere on the toast (ignoring the button) and it'd work. The info about which button is pressed will be buried in the event data.

In short... lots more documentation and examples are on the way. Sorry for the lack of it ATM!

Specifically, I'd recommend watching [PSPowerHour](https://www.twitch.tv/devopsorg) on 27 September @ 2 PM Pacific Time as I'll be demoing a lot of this sort of stuff there.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
